2020-11-11 18:55:19,070 [MainThread] INFO  main.py::setup(@Line:19) 
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
AppConfigs
  |-RootDir = /home/lucassaporetti/git_repository/frogs_tasks_organizer/task_organizer/src
  |-LogFile = /home/lucassaporetti/git_repository/frogs_tasks_organizer/task_organizer/src/../log/application.log
  |-AppProperties:
   \-persistence.repository.type=FILE
   |-persistence.database.type=FILE_STORAGET
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-= 
2020-11-11 18:55:25,792 [MainThread] DEBUG connectionpool.py::_new_conn(@Line:205) Starting new HTTP connection (1): 216.58.192.142:80 
2020-11-11 18:55:26,110 [MainThread] DEBUG connectionpool.py::_make_request(@Line:393) http://216.58.192.142:80 "GET / HTTP/1.1" 301 219 
2020-11-11 18:55:26,115 [MainThread] DEBUG connectionpool.py::_new_conn(@Line:205) Starting new HTTP connection (1): www.google.com:80 
2020-11-11 18:55:26,324 [MainThread] DEBUG connectionpool.py::_make_request(@Line:393) http://www.google.com:80 "GET / HTTP/1.1" 200 5879 
2020-11-11 18:55:26,327 [MainThread] INFO  frogs_api_repository.py::test_internet_connection(@Line:45) Internet connection: Ok 
2020-11-11 18:55:26,329 [MainThread] DEBUG connectionpool.py::_new_conn(@Line:205) Starting new HTTP connection (1): 127.0.0.1:8000 
2020-11-11 18:55:26,330 [MainThread] ERROR frogs_api_repository.py::test_api_connection(@Line:58) Error: API connection failed 
2020-11-11 18:55:26,330 [MainThread] ERROR frogs_api_repository.py::insert(@Line:73) Error: New task can't be saved 
